.github/workflows/ci.yml

```diff
@@ -61,6 +61,7 @@ jobs:
         uses: coursier/setup-action@v1
         with:
           jvm: temurin:1.17
+          apps: sbt
```

Comment:
issue: We have other jobs using sbt, which will need the same treatment. Can you add sbt in there as well?